dbux-code/src/globalAnalysisView/GlobalAnalysisViewController.js

```diff
@@ -33,18 +36,34 @@ export default class GlobalAnalysisViewController {
     this.refresh();
   }
 
+  handleSearch = async () => {
+    await this.refresh();
+    this.focusOnSearchResult();
```

Comment:
@MiccWan no `await` for this?
Since race conditions in the focus controller are our biggerst problem, can you briefly document on the expectations of the potential main culprits, such as this one?

Reply:
My bad. There should be an `await` definitely.
I add this line before actually implement the function and forget to add the missing `await` after making it an async function.